railties/lib/rails/generators/rails/plugin/templates/bin/rails.tt

```diff
@@ -9,5 +9,4 @@ APP_PATH = File.expand_path('../../<%= dummy_path -%>/config/application', __FIL
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
 
-require 'rails/all'
```

Comment:
If do not load the railtie of each component, the generator will not work properly. 
I think that we should do the same way as `application.rb`. 
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/application.rb#L3..L17